.changes/next-release/Bug Fix-fff4dbbd-2664-43c3-bf6f-98cf1440ace2.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Redshift: Fix the issue that Output panel always opens at VS Code launch."
```

- The item type is "Bug Fix" so the description just needs to describe what's being fixed.
- Prefixing Redshift changes with `Redshift:` is usually a good idea, but in this case it's a Toolkit-wide behavior that's being fixed, so we can leave out the prefix.


```suggestion
	"description": "Output panel always opens at VS Code launch."
```

---

Yes, I have made the suggested change.